dotCMS/src/integration-test/java/com/dotcms/publisher/business/PublisherTest.java

```diff
@@ -377,6 +381,23 @@ private PPBean createPushPublishEnv(final User user) throws DotSecurityException
         return new PPBean(environment, endPoint);
     }
 
+    @Test
+    //this is a test for push publish a folder with nothing inside
```

Comment:
Please use the doc  convention
Given Scenario: 
Expected Results: 

To doc what this TC does.